pipelines/rj_crm__salesforce_agentforce_api/tasks_legacy.py

```
@@ -0,0 +1,375 @@
+        ID do job criado.
+    """
+    url = f"{instance_url}/services/data/{api_version}/jobs/query"
+    headers = {
+        "Authorization": f"Bearer {access_token}",
+        "Content-Type": "application/json",
+    }
+    payload = {
+        "operation": "query",
+        "query": soql,
+    }
+    print(f"[SF_CRM][BULK] Criando job de query...")
```

Ruff (0.16.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)